src/taipy/gui/icon.py

```diff
@@ -45,14 +46,29 @@ def __init__(self, path: str, text: t.Optional[str] = None) -> None:
                 the application directory.
             text (Optional[str]): The text associated to the image. If _text_ is None,
                 there is no text associated to this image.
+            light_path (Optional[str]): The path to the light theme image (fallback to _path_ if not defined).
+            dark_path (Optional[str]): The path to the dark theme image (fallback to _path_ if not defined).
+            svg (Optional[bool]): Indicates that the _path_ should be treated as an SVG resource (defaults to True if _path_ ends with .svg).
```

Comment:
Where is this doc ("defaults to True if _path_ ends with .svg") implemented?
What if path is svg, but light_path is not?

Reply:
In [icon.tsx:57](https://github.com/Avaiga/taipy-gui/blob/feature/%23631-%23632-base-image-and-themed-icons/gui/src/utils/icon.tsx#L57)
the svg property is meant to allow non svg ending files to be treated as SVG ... Is it really necessary?

Reply:
Oh right.
And as you say... we may not even need this.